src/plugins/intel_cpu/src/nodes/subgraph.cpp

```diff
@@ -51,117 +51,166 @@ std::mutex err_print_lock;
 #include "transformations/tpp/x64/pass/lowered/set_tpp_leading_dim.hpp"
 #endif
 
-using namespace dnnl::impl::utils;
-using namespace dnnl::impl::cpu;
-using namespace dnnl::impl::cpu::x64;
```

Comment:
Minor: why did you remove the usings?
Now we have to spell `dnnl::impl::cpu::x64::mayiuse` which is longer and less readable

Reply:
After rebase on ARM PR - arm-specific jobs are failing on build stage, they cannot recognize `aarch64::mayuse()`. Also PR with ARM introduces the full namespaces there - so I left it so.